examples/arm/executor_runner/CMakeLists.txt

```diff
@@ -59,8 +59,74 @@ endif()
 
 if(TARGET_BOARD STREQUAL "corstone-300")
   add_subdirectory(${ETHOS_SDK_PATH}/core_platform/targets/corstone-300 target)
```

Comment:
since we are adding subdir first, IIRC we can't use these defs in that code. Just making sure that is OK.